File: Level-1/2.java
class Solution {
    public String solution(int a, int b) {
        String answer = "";
        String[] day = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};
        int[] month = {0, 31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        int num = 0;
        
        for(int i = 1; i < a; i++){
            num += month[i];
        }
        
        num += b;
        answer = day[(num + 4) % 7];
        
        return answer;
    }
}

File: Level-1/2.js
function solution(a, b) {
    const dayList = ['FRI', 'SAT', 'SUN', 'MON', 'TUE', 'WED', 'THU'];
    const monthArr = [31, 29, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31];
    let day = b - 1;
    
    if(a > 1) {
        for(let i = 0; i < a - 1; i++) {
            day += monthArr[i];
        }
    }
    
    return dayList[day % 7];
}
